css/styles.css

```diff
@@ -78,6 +86,426 @@
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  color: #091e42;
+}
+
+.languages {
+  grid-column: 2;
+}
+
+.languages > ul {
+  list-style-type: none;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.languages > ul > li {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 1rem;
+  line-height: 1.25rem;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+}
+
+.story-btn {
+  grid-column: 2;
+  margin-top: 1rem;
+}
+
+.story-btn > span {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.25rem;
+  line-height: 1.5rem;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+}
+
+.story-btn > span > a {
+  text-decoration: none;
+  background-color: #ff6b00;
+  color: #fff;
+  padding: 1rem;
+}
+
+.projects {
+  margin-top: 1rem;
+  display: grid;
+  grid-template-columns: 1.5rem 1fr 1.5rem;
+  grid-template-rows: 100%;
+}
+
+.project-details {
+  grid-column: 2;
+  background-image: url("../pics/img2.svg");
+  background-repeat: no-repeat;
+  padding: 0.8rem;
+}
+
+.project-title {
+  margin-top: 4.5rem;
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 2rem;
+  line-height: 2.75rem;
+  color: #fff;
+}
+
+.support-text {
+  margin-top: 1rem;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  color: #fff;
+}
+
+.lang-list > li > a {
+  margin-top: 2.75rem;
+  text-decoration: none;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1.5rem;
+  line-height: 2.25rem;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+}
+
+.lang-list {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.pro-language > ul {
+  list-style-type: none;
+  display: flex;
+}
+
+.pro-language > ul > li {
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+}
+
+.pro-btn {
+  grid-column: 2;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #ff6b00;
+}
+
+.languages > ul > li :hover {
+  background-color: #091e42;
+  color: #fff;
+}
+
+.pro-btn > span {
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+}
+
+.pro-btn > span > a {
+  text-decoration: none;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.about {
+  background-color: #1c1a19;
+  margin-top: 7.5rem;
+  display: grid;
+  grid-template-columns: 1.5rem 1fr 1.5rem;
+  grid-template-rows: 100%;
+  background-image: url("../pics/illustration-about-me-mobile.svg");
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-position-x: right;
+  background-position-y: bottom;
+}
```

Comment:
- [ ] Kindly make sure the `about` section is displayed using a flexbox as specified in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md).
    -  I noticed you're using a `grid` display instead of a `flexbox` so kindly replace that.

Reply:
done :heavy_check_mark: